diff --git a/node_modules/react-native-image-size/android/.project b/node_modules/react-native-image-size/android/.project
new file mode 100644
index 0000000..59ba9b9
--- /dev/null
+++ b/node_modules/react-native-image-size/android/.project
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>react-native-image-size</name>
+	<comment>Project react-native-image-size created by Buildship.</comment>
+	<projects>
+	</projects>
+	<buildSpec>
+		<buildCommand>
+			<name>org.eclipse.buildship.core.gradleprojectbuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+	</buildSpec>
+	<natures>
+		<nature>org.eclipse.buildship.core.gradleprojectnature</nature>
+	</natures>
+	<filteredResources>
+		<filter>
+			<id>1718685093435</id>
+			<name></name>
+			<type>30</type>
+			<matcher>
+				<id>org.eclipse.core.resources.regexFilterMatcher</id>
+				<arguments>node_modules|\.git|__CREATED_BY_JAVA_LANGUAGE_SERVER__</arguments>
+			</matcher>
+		</filter>
+	</filteredResources>
+</projectDescription>
diff --git a/node_modules/react-native-image-size/android/build.gradle b/node_modules/react-native-image-size/android/build.gradle
index bae680f..9aea71b 100644
--- a/node_modules/react-native-image-size/android/build.gradle
+++ b/node_modules/react-native-image-size/android/build.gradle
@@ -1,11 +1,12 @@
 buildscript {
     repositories {
+        google() // Add this repository
         mavenCentral()
         jcenter()
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.3'
+        classpath 'com.android.tools.build:gradle:8.0.1' // or another valid version
     }
 }
 
@@ -32,6 +33,8 @@ android {
 
 repositories {
     mavenCentral()
+    google() // Add this repository
+
 }
 
 dependencies {
